transformation/transform.py

```diff
@@ -0,0 +1,37 @@
+import cv2
+import numpy as np
+
+def transform(img):
```

Comment:
Could you add some comments? Makes it a little clearer

Reply:
👍